src/app/payments/update-pending-payments.ts

```diff
@@ -229,6 +214,30 @@ const updatePendingPayment = wrapAsyncToRunInSpan({
               return setErrorCritical(reimbursed)
             }
           }
+        } else if (status === PaymentStatus.Settled) {
```

Comment:
if we return `else` is not necessary

Reply:
Clean up here https://github.com/GaloyMoney/galoy/pull/2151/commits/242ce03ad013963d01e656030fb27b6f7739d1a1